features/clone.feature

```diff
@@ -113,6 +134,17 @@ Feature: hub clone
     Then it should clone "git@github.com:mislav/dotfiles.git"
     And there should be no output
 
+  Scenario: Clone my repo that doesn't exist
+    Given the GitHub API server:
+      """
+      get('/repos/mislav/dotfiles') { status 404 }
+      """
+    When I run `hub clone dotfiles`
+    Then the exit status should be 1
+    And the stdout should contain exactly ""
+    And the stderr should contain exactly "Error: repository mislav/dotfiles doesn't exist\n"
```

Comment:
`Error: repository git@github.com:mislav/dotfiles doesn't exist`? Do you think the full path is too confusing? If this were used on an enterprise instance, for example, it would be great to specify.